src/pages/evaluations/template.py

```diff
@@ -0,0 +1,181 @@
+import json
+
+import streamlit as st
+
+# from inspect_evals_scoring.process_log import DashboardLog
+from src.log_utils.dashboard_log import DashboardLog
+from src.log_utils.dashboard_log_utils import get_metrics
+from src.plots.bar import create_bar_chart
+from src.plots.cutoff_scatter import create_cutoff_scatter
+from src.plots.pairwise import create_pairwise_analysis_table
+from src.plots.plot_utils import highlight_confidence_intervals
+
+
+def render_page(eval_logs: list[DashboardLog]):
+    st.subheader("Naive cross-model comparison")
+    st.markdown("""
+                Graphs in this section compare how different AI models perform on the same task. We call this a "naive" comparison because it uses simple averages to compare models, which doesn't tell us if one model is statistically significantly better than another. For more reliable comparisons, check out the next section. To get more accurate scores, we evaluate each sample in a dataset multiple times using the epochs feature in Inspect AI.
+                """
+    )
+
+    col1, col2, col3, col4 = st.columns(4)
```

Comment:
The graphs currently span the full width of the screen, which can lead to them being hard to read on wide monitors. Consider a width limit

Reply:
Others suggested the same thing, we should do narrow mode by default on all pages cc. @eleweek